_assets/js/utils/paginationLogic/lastPage.es6

```diff
@@ -0,0 +1,6 @@
+const lastPage = (offsetArrayVal, offsetParamVal, resultsArray) => {
```

Comment:
This is a little persnickety, but I think these could all be in one file.

Reply:
I think that's a good idea, since these are all related.